### ISSUE TITLE
OData V2 Edm.Time is not supported

### ISSUE THREAD

**Issue by sosl-beep** [oasis-tcs/odata-openapi#302]
Hi,

Version used is `0.25.4`

When converting a OData V2 containing properties with `type="Edm.Time"` the conversion is a success however properties with `Edm.Time` are converted to 
```
"Name": {
  "nullable": true
},
```
The following is written to the console: `'Unknown type: Edm.Time'`

To replicate the following file from SAP can be used: 
[2022_FPS_02_OP_API_SALES_ORDER_SRV_0001.txt](https://github.com/user-attachments/files/16085159/2022_FPS_02_OP_API_SALES_ORDER_SRV_0001.txt)

Expected behavior is that the property is converted to:
```
"Name":{
  "type": "string",
  "nullable": true,
  "example": "PT15H51M04S"
}
```

As a test the correct OpenApi spec should match the one provided by SAP:
[2022_FPS_02_OP_API_SALES_ORDER_SRV_0001.json](https://github.com/user-attachments/files/16085276/2022_FPS_02_OP_API_SALES_ORDER_SRV_0001.json)

**Comment by ralfhandl** [comment]
@sosl-beep This is a bug in the JS-based implementation, whose support for OData V2 is incomplete.

You may want to use the XSLT-based implementation in https://github.com/oasis-tcs/odata-openapi/tree/main/tools, it produces the expected output.

**Comment by ralfhandl** [comment]
Fixed with version 0.26.0

**Comment by sosl-beep** [comment]
Hi @ralfhandl,

Again Awesome work! We really appreciate your effort on this.
